api/src/main/java/org/pmiops/workbench/api/WorkspaceController.java

```diff
@@ -100,20 +113,39 @@ private void setCdrVersionId(Workspace workspace,
     }
   }
 
+  private FirecloudWorkspaceId generateFirecloudWorkspaceId(String name) {
```

Comment:
I think it's fine not bothering there, since we have the user specified name as well.